Modules/Core/Common/test/itkShapedNeighborhoodIteratorTest.cxx

```diff
@@ -191,6 +192,21 @@ itkShapedNeighborhoodIteratorTest(int, char *[])
   }
   std::cout << it.GetPixel(off) << std::endl;
 
+  println("Testing interoperability with FlatStructuringElement");
```

Comment:
Did you also consider making it a GoogleTest unit test? Or is there a reason why GoogleTest would be less appropriate here? You know I'm a fan of GoogleTest. 😃 I think it was a great move forward for ITK 5, to start using GTest.

Reply:
Less effort was required to extend this test, than to translate this into a GTest.

Reply:
OK, but then, how can we make it easier for people to use GTest?  🤷 

I think GTest unit tests are especially useful, because they help us to describe and validate the requirements, assertions, and expectations of our components. A GTest unit test is usually also much shorter than a traditional ITK test function, making it easier to review and understand.

Reply:
We could simply add dummy GTest.cxx files for all filters, if that helps. 🤷

Reply:
A big part of the reason why extending existing tests is easier is not having to think too much about syntax, as it can be easily gleaned from the existing code. And existence of some type and variable instances makes additions easier. Dummy GTests might help a bit with getting started, but might be confusing at other times, especially to newcomers, and would definitely increase clutter. So adding them would be a mixed bag.

Reply:
- For me personally, it would help to have separate directory for GTests, next to each "test"directory, e.g., "Modules\Core\Common\gtest", next to "Modules\Core\Common\test". I asked about this before (#394), but was a very long time ago. (More than 5 years ago, actually.) Would it be worth reconsidering?

Reply:
There is going to be non-trivial amount of changes required. If you are willing to do them, I am fine with that addition.

Reply:
Thanks @dzenanz I meant just moving our existing GTest.cxx files to separate "gtest" directories (so that's not really an addition, more like a restyling). Would that be OK to you now? Because five years ago, you didn't really like the idea: https://discourse.itk.org/t/a-different-directory-for-gtest-cxx-files/1514/2 I hope you changed your mind, in the mean time 😺

Reply:
I don't mind it now 😄